### DISCUSSION TITLE
62C6E26B-D781-2C74-B0AF-74B0A06022AF

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#89] [General]
# Sound Demo

A sound demo script that comes from old Celestia. Requires Mobile Celestia 1.5.11.

https://celestia.mobi/addon?item=62C6E26B-D781-2C74-B0AF-74B0A06022AF&lang=en

**Comment by KAROLKOKAKAL** [comment]
Make it longer pls

**Reply by bird0cosmic** [reply]
What does it even sound like?

**Comment by bird0cosmic** [comment]
how does it work?
